common/src/main/scala/com/gu/zuora/ZuoraService.scala

```diff
@@ -44,6 +45,26 @@ class ZuoraService(config: ZuoraConfig, client: FutureHttpClient, baseUrl: Optio
       subscriptions <- accountIds.map(getSubscriptions).combineAll
     } yield subscriptions.find(sub => sub.hasContributorPlan(config, billingPeriod) && sub.isActive)
 
+  def getDefaultPaymentMethodId(accountNumber: String): Future[Option[String]] = {
+    val queryData = QueryData(s"select defaultPaymentMethodId from Account where AccountNumber = '$accountNumber'")
+    postJson[PaymentMethodQueryResponse](s"action/query", queryData.asJson, authHeaders)
+      .map(r => Some(r.records.head.DefaultPaymentMethodId))
+      .fallbackTo(Future(None))
```

Comment:
It is better to use `Future.successful(None)`
(https://stackoverflow.com/questions/21466652/what-is-the-difference-between-future-successfulnone-and-futurenone)